src/core/chartOptions/chartOptions.ts

```diff
@@ -0,0 +1,152 @@
+import { Options, YAxisOptions } from 'highcharts';
+import { View } from "../types/view";
+import { getAxisLabelShorteningFunction, getFormattedUnits, getToolTipFormatterFunction, getScreenReaderFormatterCallbackFunction } from './Utility/formatters';
+import Translations from '../conversion/translations';
+import { buildHighchartSeries } from './Utility/seriesDataBuilder';
+import {  getXAxisOptions } from './Utility/timeIntervals';
+import { getLinearAxisTickPositionerFunction } from './Utility/tickPositioners';
+import { defaultTheme } from "../highcharts/themes";
+
+export abstract class CommonChartOptions {
```

Comment:
These should be constants instead of classes, they all share the common highcharts options type so the classes are bit distracting here.